### PR TITLE
Experimenting with Docker

### PR DESCRIPTION
**Pull request by jhannah**
WIP. Creating PR just to have something to easily show Chris Short, in case this is useful.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#   Here's a long version: https://www.aloneguid.uk/posts/2021/05/keycloak-docker/
+#   But I found it in edge/testing, so let's use that instead?
+#     WHOOPS! This WAS here two weeks ago, no longer exists.
+#     0.688 fetch https://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
+#     So umm... just comment out for now.
+# RUN echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing' >> /etc/apk/repositories
+# RUN apk add keycloak
+# keytool      ... there's no way to see the version?
+
+COPY packages/client/next.config.js packages/client/.norse/next.config.js
```
Comment (devcshort) [inline Dockerfile]: This line and the one following should be removed. These get generated at run/build time

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
+FROM node:18-alpine
+WORKDIR /app
+
+# Do all of these first so Docker can cache the result for faster DEV cycles
+# (we can "build" (from cache) in 2 seconds instead of 5 minutes)
+COPY package.json package.json
```
Comment (devcshort) [inline Dockerfile:6]: package-lock.json should also be copied as it contains the metadata needed to make sure the correct versions are installed of all packages

File: Dockerfile (hunk 1 above)
Comment (jhannah) [reply Dockerfile]: done, thanks

File: Dockerfile (hunk 2 above)
Comment (jhannah) [reply Dockerfile:6]: done, thanks

**Comment by jhannah** [comment]
Crazy. I rebased and force pushed too many times and github got confused about what files are modified in this PR. So closing this so I can open a new one (where github works fine). See #2 instead.

**Pull request closed without merging**